src/applications/vaos/appointment-list/components/ScheduleNewAppointmentRadioButtons.jsx

```diff
@@ -26,19 +26,22 @@ import NewTabAnchor from '../../components/NewTabAnchor';
  * />
  * @module appointment-list/components
  */
-function ScheduleNewAppointmentRadioButtons({
-  canUseVaccineFlow,
-  directScheduleSettingsStatus,
-  fetchDirectScheduleSettings,
-  startNewAppointmentFlow,
-  startNewVaccineFlow,
-}) {
+export default function ScheduleNewAppointmentRadioButtons() {
   const history = useHistory();
+  const dispatch = useDispatch();
+  const canUseVaccineFlow = useSelector(
+    state => selectCanUseVaccineFlow(state),
+    shallowEqual,
```

Comment:
You don't need `shallowEqual` on this one or the one below, since neither of them are returning objects from the selector.

Reply:
Done